src/data/resources.js

```diff
@@ -549,6 +550,127 @@ const resources = [
+            },
+            {
+                name: "Beginner Track | 2D Game",
+                video: "https://www.youtube.com/watch?v=IHioQljCOFs&list=PLPO7_kXilXFbvbERRf1S0pJdXYkKPCDJr&index=5",
+                documentation: "https://drive.google.com/file/d/1Q3sIKsPwkSU2hGvSq_ivOG1km-Qyrlcb/view",
+                source: "Studio",
+                icon: studio,
+                description: "In this tutorial, we will be going over the basics of developing a simple 2D side-scroller in Unity.",
+                difficulty: "Beginner",
+            },
+            {
+                name: "Beginner Track | Tower Defense",
+                video: "https://www.youtube.com/watch?v=Y9FPlkfiEZM&list=PLPO7_kXilXFbvbERRf1S0pJdXYkKPCDJr&index=6",
+                documentation: "https://drive.google.com/file/d/1Q3sIKsPwkSU2hGvSq_ivOG1km-Qyrlcb/view",
+                source: "Studio",
+                icon: studio,
+                description: "In this tutorial, we show you how to script the mechanics for a 3D tower defense game.",
+                difficulty: "Beginner",
+            },
+            {
+                name: "Advanced Track | Shaders",
+                video: "https://www.youtube.com/watch?v=bR5q0tIIKzI&list=PLPO7_kXilXFbvbERRf1S0pJdXYkKPCDJr&index=10",
+                documentation: "https://drive.google.com/file/d/1Wd3_EqSm00f2V4PyfJ4ySb_ECJN0f5tt/view",
+                source: "Studio",
+                icon: studio,
+                description: "In this tutorial, we will introduce Unity's graphic and render pipeline. Learn how to create special effects, and manipulate lighting/texture on a code and pixel level.",
+                difficulty: "Advanced",
+            },
+            {
+                name: "Advanced Track | Particles",
+                video: "https://www.youtube.com/watch?v=bR5q0tIIKzI&list=PLPO7_kXilXFbvbERRf1S0pJdXYkKPCDJr&index=11",
+                source: "Studio",
+                icon: studio,
+                description: "In this tutorial, we'll be discussing the unity particle system and showing off some cool stuff you can achieve with it!",
+                difficulty: "Advanced",
+            },
+            {
+                name: "Advanced Track | Coroutines",
+                video: "https://www.youtube.com/watch?v=bR5q0tIIKzI&list=PLPO7_kXilXFbvbERRf1S0pJdXYkKPCDJr&index=12",
+                source: "Studio",
+                icon: studio,
+                description: "In this tutorial, we will be going over coroutines and how to build a simple state machine AI.",
+                difficulty: "Advanced",
+            },
+        ],
+    },
+    {
+        category: "Game Art",
+        links: [
+            {
+                name: "Art Tutorials | Post Processing",
+                video: "https://www.youtube.com/watch?v=CZnslTaWy5M&list=PLPO7_kXilXFbvbERRf1S0pJdXYkKPCDJr&index=7",
+                documentation: "https://drive.google.com/file/d/1L8MQb38_i0sLIP0bO0oR9hUV065iUq2y/view",
```

Comment:
For all objects here, I think we can omit the documentation links since the links themselves don't quite fall under any of the format categories we have established. For now, let's just comment them out.